src/Servers/Kestrel/samples/http2cat/Program.cs

```diff
@@ -28,7 +30,7 @@ public static async Task Main(string[] args)
 
     internal static async Task RunTestCase(Http2Utilities h2Connection)
     {
-        await h2Connection.InitializeConnectionAsync();
+        await h2Connection.InitializeConnectionAsync(expectedSettingsCount: 4);
```

Comment:
Change https://github.com/amcasey/aspnetcore/blob/bc9255b509c4b51d9bbbea7b15ddafe482784699/src/Shared/Http2cat/Http2Utilities.cs#L153 instead?

Reply:
IIS tests use Http2Utilities so we can't change the default.